src/Sulu/Bundle/SecurityBundle/Entity/User.php

```diff
@@ -306,17 +306,22 @@ public function eraseCredentials()
     public function serialize()
     {
         return \serialize(
-            [
-                $this->id,
-                $this->password,
-                $this->salt,
-                $this->username,
-                $this->locked,
-                $this->enabled,
-            ]
+            $this->__serialize()
         );
     }
 
+    public function __serialize(): array
```

Comment:
Actually we could also remove the `Serializable` interface here and remove the `serialize` and `unserialize` methods. They are in our case not longer used as we did add the `EquatableInterface` in the symfony 5 upgrade to check equality between the session serialized user and the loaded user. Removing it would be a small bc break. @sulu/core-team what do you think?

Reply:
Would be okay for me to remove the `Serializable` interface from the `User` entity 🙂

Reply:
Is removed